### ISSUE TITLE
How to resume training model with a specific weight file(from the last checkpoint file)?

### ISSUE THREAD

**Issue by GittersYang** [facebookresearch/detectron2#2106]
## ❓ How to do something using detectron2

Describe what you want to do, including:

**Comment by ppwwyyxx** [comment]
Set `cfg.MODEL.WEIGHTS` to the file, or follow https://detectron2.readthedocs.io/tutorials/models.html if you use your own training logic.

**Comment by GittersYang** [comment]
i can not understand your suggestion.  
in my eyes, i can open the  related **.yaml file**,then replace  "detectron2://ImageNetPretrained/MSRA/_R-50.pk_l" with my _.pth file._
 for example:
 WEIGHTS: "detectron2://ImageNetPretrained/MSRA/R-50.pkl"  ->_WEIGHTS: "/my/path/**xxxx04999.pth**"_ 

**and i have another issue:**

SOLVER:
  STEPS: (210000, 250000)
  MAX_ITER: 270000
How to resume training a model, rather than training so many iterations.
in other words, how to resume training from 04999  iterations(when loading xxxx04999.pth).

**Comment by ppwwyyxx** [comment]
> i can open the related .yaml file,then replace "detectron2://ImageNetPretrained/MSRA/_R-50.pk_l" with my .pth file.

yes, this is setting MODEL.WEIGHTS.

You can adjust the iteration numbers manually.
Or you can run `train_net.py` with `--resume` which can load iteration number from the output directory's "last_checkpoint" file, if exists. You can also write your own `train_net.py` for more customized behavior.

**Comment by GittersYang** [comment]
ok, thanks!!

**Comment by GittersYang** [comment]
**I can use Command line instructions:**
./train_net.py \
  --config-file ../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_1x.yaml \
  --resume=weights//my/path/xxxx04999.pth \
  --num-gpus 1 SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025

**Or i can use Command line instructions before setting MODEL.WEIGHTS:**
./train_net.py \
  --config-file ../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_1x.yaml \
  --resume \
  --num-gpus 1 SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025
